testapi.pm

```diff
@@ -267,6 +267,21 @@ sub check_var($$) {
     return 0;
 }
 
+sub get_var_array($;$) {
+    my ($var, $default) = @_;
+    my @vars = split(',|;', ($bmwqemu::vars{$var}));
+    return $default if !@vars;
+    return \@vars;
+}
+
+sub check_var_array($$) {
+    my ($var, $val) = @_;
+    my $vars_r = get_var_array($var);
+    my %vars_h = map { $_ => 1 } @{$vars_r};
+    return 1 if (defined $vars_h{$val});
+    return 0;
```

Comment:
hm, was writting it probably too much in inspiration to the 'check_var' function, no problem to simplify it like this :)